tests/blackbox/testlib/dbus.py

```diff
@@ -88,6 +89,28 @@ def _get_managed_objects():
         )
         return object_manager.GetManagedObjects(timeout=StratisDbus._TIMEOUT)
 
+    @staticmethod
+    def stratisd_version():
+        """
+        Get stratisd version
+        """
+        versions = [
+            obj_data[StratisDbus._MNGR_IFACE]["Version"]
+            for _, obj_data in StratisDbus._get_managed_objects().items()
+            if StratisDbus._MNGR_IFACE in obj_data
+        ]
+
+        return versions[0]
+
+    @staticmethod
+    def stratisd_redundancy():
+        """
+        Get stratisd redundancy
+        """
+        redundancy = "NONE = 0"
```

Comment:
Use ```return "NONE = 0"```, rather than the two lines. Also, please document why this is Ok. It very much looks like your test is a fake, since the test infrastructure is just returning its own value.